setup.cfg

```diff
@@ -0,0 +1,75 @@
+[metadata]
+name = pyAFQ
+url = https://yeatmanlab.github.io/pyAFQ
+download_url = https://github.com/yeatmanlab/pyAFQ
+author = pyAFQ developers
+author_email = arokem@gmail.com
+maintainer = Ariel Rokem
+maintainer_email = arokem@gmail.com
+classifiers =
+    Development Status :: 3 - Alpha
+    Environment :: Console
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Topic :: Scientific/Engineering
+
+license = BSD License
+description = pyAFQ: Automated Fiber Quantification ... in Python
+long_description = file:README.md
+long_description_content_type = text/markdown
+platforms = OS Independent
+
+[options]
+setup_requires =
+  setuptools_scm
+python_requires = >=3.6
+install_requires =
+    scikit_image==0.16.2
+    nibabel==3.0.2
+    dipy==1.1.1
+    scipy==1.2.0
+    numpy==1.17.5
+    pandas==1.0.3
+    joblib==0.12.4
+    dask==1.1.0
+    palettable==3.1.1
```

Comment:
Does this mean we should remove requirements.txt and requirements-*.txt, or that we need to update both when we want to change our requirements?

Reply:
Let's see. I just pushed a commit (c46b4c5) where I remove all the requirements files. I've seen other projects (e.g., nibabel) that have both the requirements files and a setup.cfg with all the requirements spelled out, so I wonder.

Reply:
Well, strangely, this installs just fine, but now can't import. I'll roll that change back. So, yes: I think that we need to remember to update in both places. At least until we can figure out why this duplication is needed.